client/components/telecom/telephony/abbreviatedNumbers/telecom-telephony-abbreviated-numbers.component.js

```diff
@@ -22,9 +22,18 @@ angular.module("managerApp").component("telecomTelephonyAbbreviatedNumbers", {
             self.loading = {
                 init: true
             };
-            self.abbreviatedNumbers = [];
+            self.abbreviatedNumbers = undefined;
         };
 
+        this.$postLink = function () {
+            self.csvButton = $element[0].querySelector(".csvButton");
```

Comment:
I think it could be better to replace that hack with
`<div role="menuitem" class="oui-action-menu__item">`
   `       <button ng-csv="..." class="oui-button oui-button_action-menu">Export</button>`
`</div>`

You are not forced to use oui-action-menu-item component for exceptions with ng-csv